internal/spentcalories/spentCalories.go

```diff
@@ -13,37 +16,72 @@ const (
 	cmInM     = 100   // количество сантиметров в метре.
 )
 
+// parseTraining возвращает данные тренировки(Шаги, продолжительность прогулки)
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// ваш код ниже
+	str := strings.Split(data, ",")
+
+	if len(str) != 3 {
+		return 0, "", 0, fmt.Errorf("Неверный формат ввода")
+	}
+
+	steps, err := strconv.Atoi(str[0])
+
+	if err != nil {
+		return 0, "", 0, fmt.Errorf("Неверные данные: %v", err)
```

Comment:
При оборачивании нужно использовать `%w`
```suggestion
		return 0, "", 0, fmt.Errorf("Неверные данные: %w", err)
```